src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null && password.equals(user.getPassword())) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
we can simplify this part of code 
```suggestion
        return user != null && password.equals(user.getPassword())
```